From b0c31f359f3345cd7a0a30dd8fa972c90d1d82f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Fri, 24 Apr 2020 15:04:18 +0300
Subject: [PATCH 2/4] contrib: ffmpeg: Remove a patch that has been merged
 upstream

This patch (with a minor modification) has been merged upstream
in http://git.videolan.org/?p=ffmpeg.git;a=commitdiff;h=08b1d1d8122517d07f2335437cde0aeedc50143f,
which was part of the 4.1 release - and we currently build
4.2.2.
---
 contrib/ffmpeg/A02-corrupt-h264-frames.patch | 17 -----------------
 1 file changed, 17 deletions(-)
 delete mode 100644 contrib/ffmpeg/A02-corrupt-h264-frames.patch

diff --git a/contrib/ffmpeg/A02-corrupt-h264-frames.patch b/contrib/ffmpeg/A02-corrupt-h264-frames.patch
deleted file mode 100644
index cf26e2063..000000000
--- a/contrib/ffmpeg/A02-corrupt-h264-frames.patch
+++ /dev/null
@@ -1,17 +0,0 @@
-diff --git a/libavcodec/h264_slice.c b/libavcodec/h264_slice.c
-index d71ddbe9ba..215a6d3c65 100644
---- a/libavcodec/h264_slice.c
-+++ b/libavcodec/h264_slice.c
-@@ -1407,6 +1407,12 @@ static int h264_field_start(H264Context *h, const H264SliceContext *sl,
- 
-     sps = h->ps.sps;
- 
-+    if (sps && sps->bitstream_restriction_flag &&
-+        h->avctx->has_b_frames < sps->num_reorder_frames) {
-+        h->avctx->has_b_frames = FFMAX(h->avctx->has_b_frames,
-+                                       sps->num_reorder_frames);
-+    }
-+
-     last_pic_droppable   = h->droppable;
-     last_pic_structure   = h->picture_structure;
-     h->droppable         = (nal->ref_idc == 0);
-- 
2.17.1

